docs/docs/examples/guides/object_detection_example.md

```diff
@@ -0,0 +1,755 @@
+---
+sidebar_position: 6
+title
+---
+
+import rvizgif from './assets/object-detection-example-rviz.gif'
+
+# Using YOLO to Track Objects
+
+This example provides a use case for `Object Detection Components`. We show how to create a custom component which
+converts bounding boxes into a 3D pose. The example works with either an Intel Realsense camera using
+`collections/intel-realsense-collection` or a video played through a custom component which is provided. The final application will be able to track an object with a robot based on a fixed camera position.
+
+<div class="text--center">
+  <img src={rvizgif} alt="Moving the robot towards an object in RVis" />
+</div>
+
+## Setup
+
+In AICA Launcher, create a configuration with the following core version and packages:
+
+- AICA Core v4.3.2
+- `collections/object-detection` (TODO: specify version)
+- `collections/intel-realsense-collection v1.0.0`
```

Comment:
TODO: use the generic camera streamer from core-vision components once they are released (@bpapaspyros)